gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/BakeService.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.services
+
+import com.netflix.spinnaker.gate.services.internal.RoscoService
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.stereotype.Component
+
+@Slf4j
+@Component
+@ConfigurationProperties('services.rosco.defaults')
+class BakeService {
+
+  @Autowired(required = false)
+  RoscoService roscoService
+
+  // Default bake options from configuration.
+  BakeOptions bakeOptions
```

Comment:
Shouldn't this be a list so that default options can be specified for multiple providers? I'm ok leaving it as simple as possible, but seems like a small change to make it more future-proof.